ansible-scylla-monitoring/tasks/common.yml

```diff
@@ -93,13 +112,13 @@
     path: "{{ base_dir }}/prometheus/prom_rules/prometheus.rules.yml"
   register: prom_rules_sub_file
 
-- name: set prometheus rules file from the default prom_rules file
+- name: copy ALL rules files from the default prom_rules file location
   copy:
-    src: "{{ base_dir }}/prometheus/prom_rules/prometheus.rules.yml"
+    src: "{{ base_dir }}/prometheus/prom_rules/*"
     remote_src: true
-    dest: "{{ scylla_monitoring_config_path }}/prometheus.rules.yml"
+    dest: "{{ scylla_monitoring_prom_rules_path }}/"
     mode: '0644'
-  when: prometheus_rules_file is defined and prometheus_rules_file == 'default' and prom_rules_sub_file.stat.exists
+  when: (prometheus_rules_file is defined) and (prometheus_rules_file != 'default') and (prom_rules_sub_file.stat.exists)
```

Comment:
And what if `prometheus_rules_file` is not defined? Don't we want to copy ALL rules files regardless?

Reply:
fixed, bug on my side, found it in testing now